test/test_helper.rb

```diff
@@ -19,3 +19,19 @@ def authorization_headers(user)
   config.cassette_library_dir = 'test/cassettes'
   config.hook_into :webmock
 end
+
+class ActiveSupport::TestCase
+  def self.test(test_name, &block)
```

Comment:
A couple things on this:

1.  Please add [this documentation](https://github.com/alexbeeken/operationcode_backend/blob/9bc9ce4b6d15aca62dcff7a52f486bacbcb30600/test/support/vcr.rb#L1-L12) to this, for future devs understanding.  Also, if you can find the url where you found this solution, please include that as an `@see ...`, like [this](https://github.com/OperationCode/operationcode_backend/blob/master/app/controllers/api/v1/users/passwords_controller.rb#L8).

2.  Please add this method to the [ActiveSupport::TestCase](https://github.com/OperationCode/operationcode_backend/blob/master/test/test_helper.rb#L8-L15) class.